app/Http/Services/Report/ReportService.php

```
@@ -2069,7 +2071,28 @@
+            $query->isActive();
+        })
+        ->with(['employee'])
+        ->get()
+        ->sortBy('employee.fullname_last', SORT_NATURAL);
+        return $data;
+    }
+    public static function userAccessibilityExport()
+    {
+        $data = self::userAccessibilityReport();
+        $fileName = "storage/temp-report-generations/UserAccessibilityReportList-" . now()->format('YmdHis');
+        $excel = SimpleExcelWriter::create($fileName . ".xlsx");
```

PHPMD (2.15.0)

Avoid using static access to class '\Spatie\SimpleExcel\SimpleExcelWriter' in method 'userAccessibilityExport'. (Clean Code Rules)

(StaticAccess)